Resources/Prototypes/Entities/Structures/Power/substation.yml

```diff
@@ -17,8 +17,11 @@
       shader: unshaded
     - state: full
       shader: unshaded
+  - type: MachineBattery
+    machinePartEfficiency: 1
```

Comment:
Why is this 1? Won't that make upgrading do nothing since `Pow(x, 1)` is just x?

Reply:
x is the PartRating, which is between 1-4 based on the parts used in this, so this just means it scales linearly

Reply:
Ah, makes sense.

Reply:
Other upgradeable machines use `base * Pow(multiplier, partRating - 1)`. Is there a reason you used a different approach here? I think it's best if they all use the same formula.

Reply:
No, that makes much more sense. Updating it.